shelley/chain-and-ledger/executable-spec/src/PParams.hs

```diff
@@ -28,7 +28,7 @@ module PParams
 
 import           Numeric.Natural (Natural)
 
-import           BaseTypes (Seed, UnitInterval, interval0, mkNonce)
+import           BaseTypes (Seed (NeutralSeed), UnitInterval, interval0)
```

Comment:
just saw that there is also `neutralseed` which just returns `NeutralSeed` in `Basetypes`. Maybe remove this en passant, I don't think there's a real use case for it.

Reply:
gone! :wave: